src/analysis/normed/group/pointwise.lean

```diff
@@ -33,6 +34,22 @@ begin
   exact norm_mul_le_of_le (hRs x hx) (hRt y hy),
 end
 
+@[to_additive] lemma metric.bounded.of_mul [has_isometric_smul E E] (hst : bounded (s * t)) :
```

Comment:
I think that we should add a lemma about `set.image2` and a map which is antilipschitz in both coordinates. Also, the conclusion seems very weak. If `s` is nonempty, then `t` is bounded.

Reply:
@eric-wieser what do you think of this comment?

Reply:
I believe I already addressed this in the subsequent commits; but am happy to wait for @urkud to take another look